<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1885</width>
    <height>1029</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>140</y>
      <width>1091</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayoutGraph" rowstretch="0,0,0">
     <item row="2" column="1">
      <widget class="QLabel" name="labelGraph2">
       <property name="text">
        <string>양품의 비율</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGraphicsView" name="graphicsViewGraph2"/>
     </item>
     <item row="1" column="0">
      <widget class="QGraphicsView" name="graphicsViewGraph1"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelGraph1">
       <property name="text">
        <string>수율</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="labelGraph3">
       <property name="text">
        <string>불량 판정 중 양품 비율</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QGraphicsView" name="graphicsViewGraph3"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>297</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutListShow" stretch="1,4">
     <item>
      <widget class="QLabel" name="labelListShow">
       <property name="text">
        <string>리스트 보기</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutListShow">
       <item>
        <widget class="QPushButton" name="pushButtonAllListShow">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>전체</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonOkListShow">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>양품</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDefListShow">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>불량품</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>310</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutControl" stretch="1,4">
     <item>
      <widget class="QLabel" name="labelControl">
       <property name="text">
        <string>제어</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutControl">
       <item>
        <widget class="QPushButton" name="pushButtonControlStart">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>시작</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonControlStop">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>정지</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>570</y>
      <width>1091</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutLog" stretch="1,9">
     <item>
      <widget class="QLabel" name="labelLog">
       <property name="text">
        <string>로그</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowserLogContent"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>871</width>
      <height>89</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutFileDir" stretch="1,0">
     <item>
      <widget class="QLabel" name="labelFileDir">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>파일 불러오기</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayoutFileDir_2" columnstretch="2,6,2">
       <item row="0" column="0">
        <widget class="QLabel" name="labelImageFileDir">
         <property name="text">
          <string>이미지셋 파일 경로</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelModelSaveFileDir">
         <property name="text">
          <string>저장 파일 경로</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="textBrowserImageFile">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="textBrowserModelSaveFile">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButtonOpenImageFile">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>이미지 파일 선택</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButtonOpenModelSaveFile">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>저장 경로 선택</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>371</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>610</y>
      <width>311</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>이미지 미리보기</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>개별 이미지 추론</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>610</y>
      <width>311</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>CAM이미지 또는 이미지 상세설명</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>880</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>추론하기</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>590</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>이미지 위치 출력</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>550</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>파일 열</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1885</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionIMAGE">
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="actionModel">
   <property name="text">
    <string>Model</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open Log File Directory</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
